claims/src/lib.rs

```diff
@@ -362,16 +359,25 @@ pub mod pallet {
 		) -> DispatchResultWithPostInfo {
 			T::MoveClaimOrigin::try_origin(origin).map(|_| ()).or_else(ensure_root)?;
 
-			Claims::<T>::take(&old).map(|c| Claims::<T>::insert(&new, c));
-			Vesting::<T>::take(&old).map(|c| Vesting::<T>::insert(&new, c));
-			Signing::<T>::take(&old).map(|c| Signing::<T>::insert(&new, c));
-			maybe_preclaim.map(|preclaim| {
+			if let Some(c) = Claims::<T>::take(&old) {
+				Claims::<T>::insert(&new, c);
+			}
```

Comment:
This one makes me curious, do you know why clippy suggests the more verbose versions?